apps/dav/lib/SystemTag/SystemTagPlugin.php

```diff
@@ -306,8 +308,11 @@ private function propfindForFile(PropFind $propFind, Node $node): void {
 			if ($user === null) {
 				return;
 			}
-	
+
 			$tags = $this->getTagsForFile($node->getId(), $user);
+			usort($tags, function (SystemTag $tagA, SystemTag $tagB): int {
```

Comment:
```suggestion
			usort($tags, function (ISystemTag $tagA, ISystemTag $tagB): int {
```
The interface has the `getName` method, why type this to the internal implementation?